Allow birthdate and country to be set from CSV (fixes #40)

dshoreman opened the pull request:
Currently you can set `dob` and `country` columns in a CSV, but nothing happens because the values aren't used and the API also overrides them.

This fixes that issue by checking for `dob` and `country` as well as replacing the hard-coded values (in the API module) with whatever's been set on the `account` object.

drallieiv commented:
Try to avoid format related changes. Otherwise all good. 
Please discard all changes concerning trailing spaces and i'll merge that.

dshoreman commented:
Apologies, my editor does that automatically. I'll reset and recommit without the space changes

dshoreman committed:
Allow setting dob and country from CSV (fixes #40)

Although the CSV parser passes all named columns through to the builder,
that method only applies username, email and password. In addition,
PtcSession within the API module forcibly overrides any set values.

This applies `dob` and `country` if they're set in the CSV, and updates
the API module to use any value set in `account` VS hardcoded values.
